CMakeLists.txt

```diff
@@ -232,6 +232,26 @@ if(WIN32)
     endif(CMAKE_SIZEOF_VOID_P EQUAL 8)
 endif(WIN32)
 
+# Setup FMA Capabilities Flag
+IF(UNIX)
+    MESSAGE(STATUS "Checking for FMA support...")
+    IF(NOT APPLE)
+        EXEC_PROGRAM(cat ARGS "/proc/cpuinfo" OUTPUT_VARIABLE CPUINFO)
```

Comment:
Do all non-Apple UNIX support `/proc/cpuinfo`? Or most of them? Or just Linux-y ones? I'm guessing this isn't supported by all non-Apple UNIX ... so, maybe `ELSEIF(LINUX)` this and then in the final `ELSE()` clause set `CPUINFO` to empty & print a warning.

Reply:
Or just set `CPUINFO` to "" up front, and in the final `ELSE()` print a warning about this OS not being supported for FMA & please consider adding in support!